### PR TITLE
I Added the Info.md hope its fine for now :D

### PR DESCRIPTION
**Pull request by BHLoneWolf0**

### CODE REVIEW

File: docs/commands/info.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+permalink: /docs/commands/info/
+---
+
+## Information Command
+
+The Info command send you a information about xf8bot, GitHub Repository, Documentation and the current prefix 
+
+**This is a level one Command.**
+
+**Usage**: `{prefix}info`
```
Comment (zAG6z) [inline docs/commands/info.md:13]:
```suggestion
## Usage
* `{prefix}info`
```

File: docs/commands/info.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+GitHub Repository: https://github.com/xf8b/xf8bot/
+
+Documentation: https://xf8b.github.io/xf8bot/docs/
```
Comment (zAG6z) [inline docs/commands/info.md:17]:
```suggestion
Documentation: https://xf8b.github.io/xf8bot/docs/

### **Author: Lone Wolf**
```

File: docs/commands/info.md (hunk 1 above)
Comment (zAG6z) [reply docs/commands/info.md:13]: not really needed but felt like it would look more clean this way

File: docs/commands/info.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Info Command
+permalink: /docs/commands/info/
+---
+
+## Information Command
+
+The Info command send you a information about xf8bot, GitHub Repository, Documentation and the current prefix
```
Comment (zAG6z) [inline docs/commands/info.md:9]:
```suggestion
The Info command tell you information about xf8bot, such as the GitHub Repository, documentation and the current prefix for this server.
```

just a little touch up

**Review by zAG6z — CHANGES_REQUESTED** [review]
depends on what xf8b thinks now but i think its pretty good :)

File: docs/commands/info.md (hunk 3 above)
Comment (xf8b) [reply docs/commands/info.md:9]: tell -> tells

**Pull request merged**